templates/base.html

```diff
@@ -49,6 +49,11 @@
         {% include '_includes/feeds.html' %}
         {% endblock feed_links %}
         {% include '_includes/analytics.html' %}
+        {% from '_includes/_defaults.html' import PHOTOS_LIGHTBOX with context %}
+        {% if PHOTOS_LIGHTBOX and article and article.photo_gallery %}
+        <!-- Gallery css -->
+        <link rel="stylesheet" href="{{ SITEURL }}/{{ THEME_STATIC_DIR }}/magnific/magnific-popup.css">
```

Comment:
Move it to article.html.

Use `head_links` block.

```
{% block head_links %}
blah blah. Iranzo your code here
{% endblock head_links %}
```

See https://github.com/Pelican-Elegant/elegant/blob/master/templates/article.html#L13-L24 for reference

Reply:
Seems that :

~~~
{% block head_links %}
{% include '_includes/photos_header.html' %}
{% endblock head_links %}
~~~ 

breaks the template and only this css is loaded not others